includes/admin/tools/logs/class-sales-logs-list-table.php

```diff
@@ -358,7 +358,7 @@ public function get_logs() {
 						'payment_id' => $payment->ID,
 						'form'       => $payment->form_id,
 						'amount'     => $payment->total,
-						'user_id'    => $payment->user_id,
+						'donor_id'    => $payment->customer_id,
```

Comment:
@DevinWalker Use `donor_id` instead of `user_id` because a donor can exist without `WP_User`